deps/v8/src/isolate.h

```diff
@@ -384,6 +384,11 @@ class Isolate {
  public:
   ~Isolate();
 
+  // ISSUE-14576: allow to access process_wide_mutex_ from sampler.cc
+  static base::LazyMutex GetProcessWideMutex() {
+    return process_wide_mutex_;
```

Comment:
Nevermind, I was just not understanding the `LazyStaticInstance` type correctly. That looks good to me.